snappy_wrappers/wrappers/pvactools/combining/environment.yaml

```diff
@@ -0,0 +1,10 @@
+channels:
+    - conda-forge
+    - bioconda
+dependencies:
+    - vcfpy
+    - numpy
+    - pyranges
+    - pandas
+    - python
+    - bcftools
```

Comment:
Please specify concrete versions, otherwise the wrapper's behaviour may not be reproducible with new dependency versions